novawallet/Common/View/TopCustomSearchView.swift

```diff
@@ -1,7 +1,7 @@
 import UIKit
 
-final class TopCustomSearchView: UIView {
-    static let preferredBarHeight: CGFloat = 48.0
+class TopCustomSearchView: UIView {
+    static let preferredBarHeight: CGFloat = 98.0
```

Comment:
I see that this value only used in the TokensManage I would just move it there to the constants otherwise it is confusing that we are changing it here but the view itself is used in many different places